internal/storage/store.go

```diff
@@ -20,6 +20,8 @@ type KVStore interface {
 	// Note that during partial failures, any successful results
 	// are discarded and an error is returned instead.
 	Get(keys ...[]byte) ([]*serverpb.KVPair, error)
+	// Delete deletes the given key.
+	Delete(key []byte) error
```

Comment:
Please include tests inside both the storage layers: `internal/storage/rocksdb/store_test.go` and `internal/storage/badger/store_test.go`

Reply:
Done